### ISSUE TITLE
Mesh picking example (simple_picking) fails (runs incorrectly)

### ISSUE THREAD

**Issue by s0lst1ce** [bevyengine/bevy#16865]
## Bevy version
I am using bevy 0.15.0

## Relevant system information

I am using Arch linux without DE (QTile is my WM). I use the X server and proprietary nvidia drivers.

Here is the adapter info.

```ignore
AdapterInfo { name: "NVIDIA GeForce GTX 1080", vendor: 4318, device: 7040, device_type: DiscreteGpu, driver: "NVIDIA", driver_info: "565.77", backend: Vulkan }
```

## What you did

I tried running the `simple_picking` example.

## What went wrong

While the window does open it's blank. No text appears and clicking where it should be doesn't do anything visible (be it in the terminal or the window). It should have from my understanding of the code.

I do get this warning
```
WARN bevy_render::view::window: Couldn't get swap chain texture after configuring. Cause: 'The underlying surface has changed, and therefore the swap chain must be updated'
```

Running other 3D examples work. The `3d_scene` one doesn't even produce the warning. The `transparency_3d` works and produces the same warning. I haven't tested other examples.

## Additional information

I have been discussing this with @alice-i-cecile on [discord](https://discord.com/channels/691052431525675048/1318675671446454374)

**Comment by Henauxg** [comment]
I don't think this would relate to picking, the `MeshPickingPlugin` does not initialize anything related to rendering.

Maybe it has more to do with text rendering ? There's no text in either `3d_scene` or `transparency_3d`. Would the problem still happen without the text spawning in `simple_picking` ?

**Comment by s0lst1ce** [comment]
The issue has evolved some now (bevy 0.16). Everything now appears to be working correctly, but I still get the error. I will look up what you're suggesting @Henauxg

**Comment by alice-i-cecile** [comment]
I'm going to mark this as fixed then; the swapchain warning mentioned in the original comment is relatively normal and harmless.

**Comment by s0lst1ce** [comment]
I'm fine with this being marked as fixed, it's quite true after all. Still, I'd be curious to know why this was caused, do you have ideas ? Also, if so normal why is it a warning ?

**Comment by alice-i-cecile** [comment]
With respect to it not rendering; no idea!

With regards to the warning, I believe that the way we handle swapchains on some OS's is slightly wrong, resulting in this warning. It's not critical, so no one has dug in and fixed it yet.
